docs/source/markdown/podman-volume-ls.1.md.in

```diff
@@ -32,6 +32,29 @@ Volumes can be filtered by the following attributes:
 
 Format volume output using Go template.
 
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder**           | **Description**                              |
+| ------------------------- | -------------------------------------------- |
+| .Anonymous                | Indicates whether volume is anonymous        |
+| .CreatedAt                | Volume creation time                         |
+| .Driver                   | Volume driver                                |
+| .GID                      | GID to create volume with                    |
```

Comment:
Maybe "GID of volume"? "to create with" seems more applicable to `volume create` than `ls`.